examples/MyModule/configure.ac

```diff
@@ -0,0 +1,190 @@
+AC_PREREQ(2.52)
```

Comment:
In configure.ac we require 2.63. I think we should stick to one version across NEST?